src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/boss_archimonde.cpp

```diff
@@ -531,6 +588,7 @@ class boss_archimonde : public CreatureScript
         uint32 _unleashSpell;
         bool Enraged;
         bool HasProtected;
+        std::list<Unit*> spellProtectionOfEluneTargets;
```

Comment:
Storing Unit* is the most common way for scripts to cause a crash when the Unit* is deleted, leaving a pointer to free'd memory